### PR TITLE
Allow conversion for different domains and rails engines

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

File: lib/gettext_i18n_rails_js/task.rb [hunk 1]
```diff
@@ -103,11 +103,19 @@ def files_list
     end
 
     def output_path
-      ::Rails.root.join(
+      rails_root.join(
         GettextI18nRailsJs.config.output_path
       )
     end
 
+    def rails_root
```
Comment (ekohl) [inline lib/gettext_i18n_rails_js/task.rb:111]: Should this be `engine_root`?

Comment (tboerger) [reply lib/gettext_i18n_rails_js/task.rb]: Do you want to rename it or keep it like that?

**Comment by tboerger** [comment]
Please provide a changelog entry :)

**Comment by tboerger** [comment]
I have released 1.4.0 now.

**Pull request merged**